package.json

```diff
@@ -20,6 +20,7 @@
     "libsignal": "1.2.1",
     "long": "3.2.0",
     "node-fetch": "1.7.3",
+    "pg": "^7.4.1",
```

Comment:
We should look at making this and the redis dep optional so simple installs don't have to have it perse.  I think NPM has a facility for that.